src/vita-make-fself.c

```diff
@@ -171,23 +178,36 @@ int main(int argc, const char **argv) {
 	ver.unk3 = 16;
 	ver.unk4 = 0;
 
-	SCE_controlinfo_5 control_5 = { 0 };
-	control_5.common.type = 5;
-	control_5.common.size = sizeof(control_5);
-	control_5.common.unk = 1;
-	SCE_controlinfo_6 control_6 = { 0 };
-	control_6.common.type = 6;
-	control_6.common.size = sizeof(control_6);
-	control_6.common.unk = 1;
-	control_6.is_used = 1;
```

Comment:
removed this line.
is it safe? if that, please update the comment of struct definition; `/* always set to 1 */`